### DISCUSSION TITLE
Should css.square.small be the owner avatar for CSS-Next?

### DISCUSSION THREAD

**Discussion by tylerjmorg** [CSS-Next/logo.css#56] [General]
I was looking on [w3c/csswg-drafts#11193](https://github.com/w3c/csswg-drafts/issues/11193) and some good points were brought up regarding the logo when it becomes too small.

The owner avatar on repos is rendered as 24x24:
<img width="351" alt="Screenshot 2024-12-25 at 7 51 15 PM" src="https://github.com/user-attachments/assets/3f6976d9-60cb-4713-903b-10cd3d54fde9" />
<img width="332" alt="Screenshot 2024-12-25 at 7 51 25 PM" src="https://github.com/user-attachments/assets/f28a755c-78f3-4ef5-a596-4d42d9869c18" />

This may be a great use case to switch to css.square.small. 

However, it may cause confusion when a user is on the CSS-Next profile since it's rendered 100x100:
<img width="359" alt="Screenshot 2024-12-25 at 7 53 55 PM" src="https://github.com/user-attachments/assets/171937b6-8803-4030-89bc-bc58ca895ce4" />
<img width="421" alt="Screenshot 2024-12-25 at 7 55 32 PM" src="https://github.com/user-attachments/assets/24ec268e-930f-4934-8431-5323e0749074" />

Would it be worth switching the owner avatar to css.square.small for the sake of accessibility?

**Comment by argyleink** [comment]
hi, thanks for the comments and concern! iiuc, this is a petition for the CSS Next github org avatar photo to use the variant that is optimized for < 40px scenarios? 

I think this train of thought makes sense, but I'm stopped by my inability to find anywhere that the organization avatar is rendered that small, so i'm not sure it's an actual issue and is instead a theoretical issue? I'm also not sure there's an accessibility issue here, you don't need to be able to read the letters in an avatar photo everywhere, easy to go to a profile page and see it slightly larger if you someone really wanted to read the letters (and not the alt). the more I think about it, the less i'm convinced we should change the logo here. but maybe you want to share a counter argument that like, the majority of cases folks see this org logo they see it small, or something?

I can also bring it up at the next community group meeting and see what the rest of the folks think 👍🏻
